docs/linux-requirements.md

```diff
@@ -53,7 +53,7 @@ Remember that if you are using the `root` user, it is not necessary to use `sudo
 **Example:**
 
 ```sh
-apt-get update && apt-get install git cmake make gcc g++ clang default-libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mariadb-server libace-6.* libace-dev -y
+apt-get update && apt-get install apt-get install git cmake make gcc g++ clang default-libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libncurses-dev mariadb-server libboost-all-dev -y
```

Comment:
there is a repetition: `apt-get install apt-get install`

Reply:
> 

Type error, copy paste oriented programming